JS-Drum-Kit/app.js

```diff
@@ -0,0 +1,32 @@
+const keys = document.querySelectorAll('.key');
+
+// Play drum by keyboard press
+
+function playDrum(e) {
```

Comment:
and change the function to
```suggestion
function playDrum(e) {
play(e.keyCode)
}
```

Reply:
done